eslint_rules/src/rules/dto-and-entity-style.test.ts

```diff
@@ -44,5 +24,47 @@ ruleTester.run("definite-or-optional", rule, {
         },
       ],
     },
+    {
+      code: `class FooDto { a!: number}`,
+      errors: [
+        {
+          column: 1,
+          endColumn: 27,
+          line: 1,
+          endLine: 1,
```

Comment:
Had you mentioned these were hard to maintain?

Reply:
I'll get rid of this in a follow up.